src/resolvers/registries/npm-resolver.js

```diff
@@ -24,6 +24,10 @@ export default class NpmResolver extends RegistryResolver {
   static registry = 'npm';
 
   static async findVersionInRegistryResponse(config: Config, range: string, body: RegistryResponse): Promise<Manifest> {
+    if (!body['dist-tags']) {
+      throw new MessageError(`Received malformed response from registry. The registry may be down.`);
```

Comment:
This needs to use `this.reporter.lang`, look for other uses in the codebase to see what I mean